serving/docs/modes.md

```diff
@@ -182,7 +182,7 @@ To get this model, clone the djl-demo repo if you haven't done so yet. Then, we
 ```
 git clone https://github.com/deepjavalibrary/djl-demo.git
 cd djl-demo/djl-serving/python-mode/resnet18
```

Comment:
```suggestion
cd djl-demo/djl-serving/python-mode/
zip -r resnet18.zip resnet
```

Reply:
We should keep the zip file name as resnet18.zip

Reply:
Changed.